src/EnergyPlus/ReportSizingManager.cc

```diff
@@ -1279,6 +1279,11 @@ namespace ReportSizingManager {
                         }
                     }
                 } else if (SizingType == CoolingWaterflowSizing) {
+                    if (ZoneEqOutdoorAirUnit > 0) {
+                        CoilDesWaterDeltaT = 0.5 * DataWaterCoilSizCoolDeltaT;
```

Comment:
@Nigusse Why the 0.5 for outdoor air unit coils?  @rraustad How could this have been missing for so long? Wouldn't ChW fancoil sizing, for example, have failed?

Reply:
@mjwitte That is what we do when cooling coils are on 100%OA stream such as for coils on OA system. OutdoorAirUnt is for 100% OA system, and I am assuming the same logic here as well.

Reply:
@rraustad Why the 0.5? Seems arbitrary, will result in oversized plant water flow rate.

Reply:
This variable IS initialized in my Refactor Unitary System branch and I see it in other branches from a month ago so my guess is that someone mistakenly deleted this line lately which would make sense. The implementation for air loop equipment is:

    if (CurOASysNum > 0) {
        CoilDesWaterDeltaT = 0.5 * DataWaterCoilSizCoolDeltaT;
    } else {
        CoilDesWaterDeltaT = DataWaterCoilSizCoolDeltaT;
    }

This isn't what "we" do, it is what the developer did. This is wrong and should be corrected as a separate issue.

Reply:
It happened [here](https://github.com/NREL/EnergyPlus/commit/fb7543800d9a4f53560fdb3c9e4d9209c4fe1a1f#diff-de781ad7ec80e690e175811f4da40a5aL1282) in a merge conflict resolution. We need to get that back in place sooner than waiting for the OA unit PR to get reviewed and merged.

Reply:
Heating coils in the OA systems are sized directly using plant delta T

    } else if (SizingType == HeatingWaterflowSizing) {
        ..snip..
        AutosizeDes = DesCoilLoad / (DataWaterCoilSizHeatDeltaT * Cp * rho);

Reply:
@Nigusse change this back the way is was. Which is just:

    CoilDesWaterDeltaT = DataWaterCoilSizCoolDeltaT;

Reply:
@mjwitte do you want me to push a quick change or are you doing that now?

Reply:
@rraustad If you would, please.  Thanks.  Also maybe diff ReportSizingManager with one of your branches to make sure all the changes in it are legit.

Reply:
Created new branch and diff'ed with my Refactor UnitarySystem branch. Only missing code is this line. There is new code which looks like something I did recently. Will push this one line code change soon.

This is new:

    if (TermUnitSizing(CurTermUnitSizingNum).InducesPlenumAir) {
        AutosizeDes = (TermUnitFinalZoneSizing(CurTermUnitSizingNum).DesHeatCoilInTempTU * MinFlowFrac) +
                                          (TermUnitFinalZoneSizing(CurTermUnitSizingNum).ZoneRetTempAtHeatPeak * (1.0 - MinFlowFrac));
    } else {
        AutosizeDes = TermUnitFinalZoneSizing(CurTermUnitSizingNum).DesHeatCoilInTempTU * MinFlowFrac +
                                          FinalZoneSizing(CurZoneEqNum).ZoneTempAtHeatPeak * (1.0 - MinFlowFrac);
    }